wiki/Guides/How_to_Use_the_Offset_Wizard/en.md

```diff
@@ -1,3 +1,25 @@
 # How to use the offset wizard
+*See also: [Universal Offset](/wiki/Universal_offset)*
 
-This guide was incorrect and has been removed.
+If the [Hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **offset wizard**.
```

Comment:
```suggestion
If the [Hitsounds](/wiki/Beatmapping/Hitsound) felt slightly out of sync with the music, in every beatmap that you've played, you will more than likely need to change your [Universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **offset wizard**.
```

Reply:
I think the tense is fine as-is?

Reply:
Feels wrong changing to past tense so will leave as is. Fixing everything else though as suggested. Thanks for the help :)

Reply:
Yeah, it is fine. But just wanted to make it to sound a little bit nicer (i think?)